libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -143,14 +152,18 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
     override suspend fun setRoute(
         route: DirectionsRoute?,
         legIndex: Int
-    ): NavigationStatus =
+    ): Boolean =
         withContext(NavigatorDispatcher) {
             MapboxNativeNavigatorImpl.route = route
             val result = navigator!!.setRoute(
                 route?.toJson()
                     ?: "{}",
                 PRIMARY_ROUTE_INDEX,
-                legIndex
+                legIndex,
+                ActiveGuidanceOptions(
+                    ActiveGuidanceMode.KDRIVING,
+                    ActiveGuidanceGeometryEncoding.KPOLYLINE6
+                )
```

Comment:
New `ActiveGuidanceOptions` @kmadsen

Reply:
+1

I was thinking we would need something different. But actually, we may map this directly from the DirectionsCriteria

Reply:
We'll add a 

```
ActiveGuidanceOptionsMapper.mapFrom(directionsRoute: DirectionsRoute)
```